index.html

```diff
@@ -268,6 +268,72 @@ <h4 class="experience-title">Skills</h4>
         </li>
       </ul>
     </section>
+    <section class="contact-form">
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </p>
+      <form class="form" action="https://formspree.io/f/xknkyppy" method="POST">
+        <input
+          type="text"
+          name="name"
+          maxlength="30"
+          placeholder="Full name"
+          required
+        />
+        <input type="email" name="email" placeholder="Email address" required />
+        <textarea maxlength="500" name="message" required></textarea>
```

Comment:
[OPTIONAL]
- [ ] It would be better if you also add a placeholder text to this field, because it's really hard to determine what this field is for since the design doesn't have any labels.

This is a requirement of the project but first reviewer didn't notice it so I requested this as an optional but I **highly** recommend you to make the change.